blueman/main/DbusService.py

```diff
@@ -1,28 +1,28 @@
 import logging
 import sys
 import traceback
-from typing import Dict, Tuple, Callable, Set
+from typing import Dict, Tuple, Callable, Set, Optional, Any, Collection
 
 from gi.repository import Gio, GLib
 
 
 class DbusError(Exception):
     _name = "org.blueman.Error"
 
-    def __init__(self, message=None):
+    def __init__(self, message: Optional[str] = None) -> None:
         self._message = message
 
     @property
-    def name(self):
+    def name(self) -> str:
         return self._name
 
     @property
-    def message(self):
+    def message(self) -> Optional[str]:
         return self._message
 
 
 class DbusService:
-    def __init__(self, bus_name, interface_name, path, bus_type):
+    def __init__(self, bus_name: Optional[str], interface_name: str, path: str, bus_type: Gio.BusType) -> None:
```

Comment:
bus_name should never be optional as there is no point in having a dbus service but not own a name.

Reply:
There is. We use it for our agents which we then just register (by object path). They do not need to own a specific bus name.